external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutTopicPartition.java

```diff
@@ -43,7 +43,7 @@ public TopicPartition getTopicPartition() {
 
     @Override
     public String getId() {
-        return topicPartition.topic() + "/" + topicPartition.partition();
+        return topicPartition.topic() + "_" + topicPartition.partition();
```

Comment:
so which one do you think it should be? This is new code that I wrote, so it should be a matter of choice. Before this change I had **/** but I felt that there was no need to create a new zookeeper level, hence I changed it to **_**. **_** is what exists in the old kafka consumer code. Please take a look at method `org.apache.storm.kafka.Partition#getId()` and let me know which one we should go with. Currently the code works using either **/** or **_**